### PR TITLE
Feature/change view

### PR DESCRIPTION
**Pull request by oricc**
Added an option to wrap the results and join them using a string

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -17,9 +17,9 @@
   background-color: #282c34;
   min-height: 10vh;
   display: flex;
-  flex-direction: column;
+  flex-direction: row;
```
Comment (Kovler10) [inline src/App.css]: ```display: flex;``` has ```flex-direction: row;``` by default. so you can delete this line

File: src/components/ListFormatControls/ListFormatControls.jsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import { useListContext } from "../../context/ListContext";
+import "./ListFormatControls.css";
+const ListFormatControls = () => {
+  const { delimiter, setDelimiter, wrapper, setWrapper } = useListContext();
+
+  return (
+    <div className="input-box-controls">
+      <label className="input-label">Wrapper: </label>
+      <input
+        className="input-text"
```
Comment (Kovler10) [inline src/components/ListFormatControls/ListFormatControls.jsx:10]: wrap props with ```{}```

File: src/components/ListArea/ListArea.jsx [hunk 3]
```diff
@@ -4,20 +4,24 @@ import { useListContext } from "../../context/ListContext.jsx";
...
   const [value, setValue] = useState();
-  const { listA, listB, setListByIndex, getListByIndex } = useListContext();
+
+  const { listA, listB, setListByIndex, getListByIndex , delimiter, wrapper} = useListContext();
 
   useEffect(() => {
     if (readonly) {
-      const displayList = getListByIndex(listIndex).join("\n");
+      const displayList = getListByIndex(listIndex)
+        .map((value) => wrapper + value + wrapper)
```
Comment (Kovler10) [inline src/components/ListArea/ListArea.jsx]: use Template literals here instead of ```+```

**Review by Kovler10 — CHANGES_REQUESTED** [review]

File: src/components/ListFormatControls/ListFormatControls.jsx (hunk 2 above)
Comment (oricc) [reply src/components/ListFormatControls/ListFormatControls.jsx:10]: What props? This is an HTML input element

**Pull request merged**